pkgs/development/tools/mold/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, lib, clang_12, cmake, lld_12, tbb, xxHash, zlib, openssl, git }:
+stdenv.mkDerivation rec {
+  pname = "mold";
+  version = "0.9.1";
+  src = fetchFromGitHub {
+    owner = "rui314";
+    repo = "mold";
+    rev = "v${version}";
+    sha256 = "sha256-yIkW6OCXhlHZ1jC8/yMAdJbSgY9K40POT2zWv6wYr5E=";
+  };
+  nativeBuildInputs = [ clang_12 cmake lld_12 tbb xxHash zlib openssl git ];
```

Comment:
tbb, zlib and openssl are build time library dependencies, they should probably be moved to buildInputs.

Reply:
tbb, zlib and openssl are ~~build~~ *run* time library dependencies. I moved them to buildInputs, thanks.